### PR TITLE
Np update reference inputs PD-2223

### PR DESCRIPTION
**Pull request by nikellepetrillo**
### Description

We have copied up some of our reference files to the google public references bucket. This PR is updating the our references to point to the public bucket. 

----

### Checklist 
If you can answer "yes" to the following items, please add a checkmark next to the appropriate checklist item(s) **and** notify our WARP documentation team by tagging either @ekiernan or @kayleemathews in a comment on this PR.

- [ ] Did you add inputs, outputs, or tasks to a workflow?
- [ ] Did you modify, delete or move: file paths, file names, input names, output names, or task names?
- [ ] If you made a changelog update, did you update the pipeline version number?

### CODE REVIEW

**Comment by nikellepetrillo** [comment]
scientific test running here: https://gotc-jenkins.dsp-techops.broadinstitute.org/job/warp-workflow-tests/50442/

File: pipelines/skylab/optimus/test_inputs/Plumbing/mouse_v2_example.json [hunk 1]
```diff
@@ -23,10 +23,10 @@
...
       "gs://broad-gotc-test-storage/Optimus/input/plumbing/chemistry_X10_V2/neuron6k_mouse_chr19_genic/neurons_2000_S1_L006_I1_001.filtered.fastq.gz"
   ],
-  "Optimus.tar_star_reference": "gs://gcp-public-data--broad-references/mm10/v0/star/star_2.7.9a_primary_gencode_mouse_vM21.tar",
+  "Optimus.tar_star_reference": "gs://gcp-public-data--broad-references/GRCm39/star/v2_7_10a/modified_star2.7.10a-Mouse-GENCODE-build-GRCm39-M32.tar",
   "Optimus.input_id": "neurons2k_mouse",
   "Optimus.tenx_chemistry_version": "2",
   "Optimus.star_strand_mode": "Unstranded",
-  "Optimus.annotations_gtf": "gs://gcp-public-data--broad-references/mm10/v0/gencode.vM21.primary_assembly.annotation.gtf",
+  "Optimus.annotations_gtf": "gs://gcp-public-data--broad-references/GRCm39/star/v2_7_10a/modified_vM32.annotation.gtf",
   "Optimus.ref_genome_fasta": "gs://gcp-public-data--broad-references/mm10/v0/GRCm38.primary_assembly.genome.fa"
```
Comment (nikellepetrillo) [inline pipelines/skylab/optimus/test_inputs/Plumbing/mouse_v2_example.json:31]: need to update this to GRCm39 (but it is not in public bucket )

File: pipelines/skylab/optimus/test_inputs/Plumbing/mouse_v2_snRNA_example.json [hunk 2]
```diff
@@ -19,11 +19,11 @@
...
     "gs://broad-gotc-test-storage/Optimus/input/plumbing/mouse_v2/nuclei_2k_S1_L008_R2_001.fastq.gz"
   ],
-  "Optimus.tar_star_reference": "gs://gcp-public-data--broad-references/mm10/v0/star/star_2.7.9a_primary_gencode_mouse_vM21.tar",
+  "Optimus.tar_star_reference": "gs://gcp-public-data--broad-references/GRCm39/star/v2_7_10a/modified_star2.7.10a-Mouse-GENCODE-build-GRCm39-M32.tar",
   "Optimus.input_id": "nuclei_2k_mouse",
   "Optimus.tenx_chemistry_version": "2",
   "Optimus.star_strand_mode": "Unstranded",
-  "Optimus.annotations_gtf": "gs://gcp-public-data--broad-references/mm10/v0/gencode.vM21.primary_assembly.annotation.gtf",
+  "Optimus.annotations_gtf": "gs://gcp-public-data--broad-references/GRCm39/star/v2_7_10a/modified_vM32.annotation.gtf",
   "Optimus.ref_genome_fasta": "gs://gcp-public-data--broad-references/mm10/v0/GRCm38.primary_assembly.genome.fa",
```
Comment (nikellepetrillo) [inline pipelines/skylab/optimus/test_inputs/Plumbing/mouse_v2_snRNA_example.json:27]: need to update this to GRCm39 (but it is not in public bucket )

Comment (ekiernan) [reply pipelines/skylab/optimus/test_inputs/Plumbing/mouse_v2_snRNA_example.json:27]: @nikellepetrillo the GRCm39 v32 was on the list you sent. Is it still not in the bucket?

**Review by ekiernan — APPROVED** [review]

File: pipelines/skylab/optimus/test_inputs/Plumbing/mouse_v2_snRNA_example.json (hunk 2 above)
Comment (nikellepetrillo) [reply pipelines/skylab/optimus/test_inputs/Plumbing/mouse_v2_snRNA_example.json:27]: @ekiernan looks like i commented on the wrong line! I mean to make this comment on line 27 (for the ref_genome_fasta)

**Review by khajoue2 — APPROVED** [review]

File: pipelines/skylab/optimus/test_inputs/Plumbing/mouse_v2_example.json (hunk 1 above)
Comment (jessicaway) [reply pipelines/skylab/optimus/test_inputs/Plumbing/mouse_v2_example.json:31]: is this a file generated by the build indices pipeline or just one we use directly?

**Review by jessicaway — APPROVED** [review]
Should we wait for the last file to get to the public bucket?

**Pull request merged**